packages/remix-docs-gen/src/export.ts

```diff
@@ -1,20 +1,55 @@
 import chalk from "chalk";
 import fs from "fs-extra";
 import * as path from "path";
-import { ComputedRoute } from "./types";
+import { ComputedRoute, Route, RouteDocsAggregate } from "./types";
 import { logSuccess } from "./utils";
 
+const generateDocsForRoutes = (routes: ComputedRoute[]) => {
+  return routes.map((item) => `"${item.path}": ${item.docs},`).join("\n    ");
+};
+
 export const exportDocs = (routes: ComputedRoute[], outputPath: string) => {
-  const routeDocs = routes
-    .map((item) => `"${item.path}": ${item.docs},`)
-    .join("\n    ");
+  const routeDocsAggregate = routes.reduce<RouteDocsAggregate>(
```

Comment:
I basically just wanted to say "Build an all routes array, a loaders array, and an actions array" then with those we can easily just create three seperate types.